doc/introduction/interfaces.rst

```diff
@@ -236,13 +236,123 @@ support gradients of QNodes.
++                  +------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+|                  | ``"finite-diff"``                  |    :rd:`2`   |     :rd:`2`   |    :rd:`2`   | :rd:`2`      | :rd:`2`       |   :rd:`2`      | :rd:`2`        |:rd:`2`      |
++------------------+------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+| ``"autograd"``   | ``"device"``                       |  :rd:`3`     |      :rd:`3`  |    :rd:`3`   | :rd:`3`      |   :rd:`3`     |  :rd:`3`       |   :rd:`3`      | :rd:`3`     |
++                  +------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+|                  | ``"backprop"``                     |     :gr:`4`  |   :gr:`4`     |     :gr:`5`  |     :rd:`9`  |   :gr:`5`     |    :gr:`5`     |   :gr:`5`      | :gr:`5`     |
++                  +------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+|                  | ``"adjoint"``                      |      :rd:`6` |     :rd:`6`   |  :rd:`6`     | :rd:`6`      |      :gr:`7`  |  :gr:`7`       |   :gr:`7`      | :rd:`6`     |
++                  +------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+|                  | ``"parameter-shift"``              |   :rd:`10`   |    :rd:`10`   |   :gr:`8`    |  :rd:`9`     |   :gr:`8`     |   :gr:`8`      | :gr:`8`        |   :gr:`8`   |
++                  +------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+|                  | ``"finite-diff"``                  |   :rd:`10`   |    :rd:`10`   |   :gr:`8`    |  :rd:`9`     |   :gr:`8`     |   :gr:`8`      | :gr:`8`        |   :gr:`8`   |
++------------------+------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+| ``"jax"``        | ``"device"``                       |  :rd:`3`     |      :rd:`3`  |    :rd:`3`   | :rd:`3`      |   :rd:`3`     |  :rd:`3`       |   :rd:`3`      | :rd:`3`     |
++                  +------------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+
+|                  | ``"backprop"``                     |     :gr:`4`  |   :gr:`4`     |     :gr:`5`  |     :rd:`9`  |   :gr:`5`     |    :gr:`5`     |   :gr:`5`      | :gr:`5`     |
```

Comment:
I think you changed the `qml.probs` column instead of `qml.state` by mistake. But yes, I agree that the cells for `qml.state` and `qml.density_matrix` for JAX, TF, and torch should now be "5".

Reply:
Changed to show that differentiating states directly is possible with JAX, TF, and torch.